internal/settings/manager.go

```diff
@@ -67,6 +67,10 @@ func (m *Manager) GetPullRequestRequeueDuration(ctx context.Context) (time.Durat
 	return controllerConfiguration.Spec.PullRequestRequeueDuration.Duration, nil
 }
 
+func (m *Manager) GetGlobalNamespace() string {
+	return m.config.GlobalNamespace
+}
```

Comment:
Can we go ahead and rename this `ControllerNamespace` across the board, just to be more explicit?

And maybe make the field private so that no one can accidentally overwrite the value after the Manager has been initialized.